packages/sync/src/modules/Module.php

```diff
@@ -33,6 +34,28 @@ abstract class Module {
 	 */
 	abstract public function name();
 
+	/**
+	 * The id field in the database.
+	 *
+	 * @access public
+	 *
+	 * @return string
+	 */
+	public function id_field() {
+		return 'ID';
+	}
+
+	/**
+	 * The table in the database.
+	 *
+	 * @access public
+	 *
+	 * @return string|bool
+	 */
+	public function table_name() {
+		return false;
```

Comment:
Should this be an abstract method instead? I think it's a better way to protect against a non-defined table name in the inheriting classes.

Reply:
Then i'll have write the same in every module. I think having `false` is a sensible default here.

Reply:
While I agree on your suggestion to use a default and to not enforce the inheriting class to implement their own `table_name()`, I think that we could reduce the code by using a class property instead of a method. What do you think?